docs/maya/tools.md

```diff
@@ -120,6 +120,8 @@ In the specific case of an AdnSkin deformer, the tool will display the following
     - This value is normalized alongside Soft Constraints and Slide Constraints.
   - **Masses**
     - Masses are set to 1.0 by default. This will mean that by default the solver will consider that the skin has a uniform mass.
+  - **Shape Preservation**
```

Comment:
I think we need to add the Shape Preservation also to the muscle deformers section, updating the paint tool screenshot.

Reply:
Done!